src/combinatorics/simple_graph/basic.lean

```diff
@@ -572,6 +572,54 @@ begin
   tauto,
 end
 
+lemma compl_neighbor_set (G : simple_graph V) (v : V) :
+  Gᶜ.neighbor_set v = (G.neighbor_set v)ᶜ \ {v} :=
+begin
+  ext w,
+  simp,
+  exact ⟨λ ⟨hne, hnadj⟩, ⟨hnadj, ne.symm hne⟩, λ ⟨hnadj, hne⟩, ⟨ne.symm hne, hnadj⟩⟩,
+end
+
+variables [decidable_eq V]
+
+lemma neighbor_set_union_compl_neighbor_set_card [fintype V] (G : simple_graph V)
+  [decidable_rel G.adj] [decidable_rel Gᶜ.adj] (v : V) :
+  fintype.card ((G.neighbor_set v ∪ Gᶜ.neighbor_set v) : set V) = fintype.card V - 1 :=
+begin
+  -- i don't love the `fintype.subtype_of_fintype situation` i have here
+  classical,
+  simp_rw neighbor_set_union_compl_neighbor_set_eq,
+  rw ← set.to_finset_card,
+  simp,
+  rw fintype.card_of_finset' {v}ᶜ,
+  rw card_compl,
+  rw ← card_singleton v,
+  simp,
+end
+
+lemma card_compl_neighbor_set [fintype V] (G : simple_graph V) [decidable_rel G.adj] (v : V) :
+  Gᶜ.degree v = fintype.card V - G.degree v - 1 :=
```

Comment:
I changed it to be in the `fintype.card V - 1 - G.degree v` form. This makes sense because `fintype.card V - 1` is the maximum possible degree, and we're subtracting `G.degree v` from that.